recipes/eternal/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.layout import basic_layout
+import os
+
+
+required_conan_version = ">=1.51.3"
+
+
+class EternalConan(ConanFile):
+    name = "eternal"
+    description = "A C++14 compile-time/constexpr map and hash map with minimal binary footprint"
+    license = "ISC"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mapbox/eternal"
+    topics = ("hashing", "map", "constexpr", "header-only")
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 14
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "15",
+            "msvc": "1910",
```

Comment:
I thought this was 3 digits 🤔 https://github.com/conan-io/conan-center-index/blob/a932b2aa205b9aebfc728fee3914675134e02eb0/recipes/argparse/all/conanfile.py#L34

Reply:
@prince-chrismc 
I misunderstanding msvc version.
Thank you.

Reply:
fixed